package.json

```diff
@@ -47,13 +47,13 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-native": "0.68.2",
-    "react-native-color-palette": "^2.2.0",
     "react-native-calendar-picker": "~7.1.2",
+    "react-native-color-palette": "^2.2.0",
     "react-native-gesture-handler": "~2.2.1",
     "react-native-phone-number-input": "^2.1.0",
     "react-native-safe-area-context": "4.2.4",
     "react-native-screens": "~3.11.1",
-    "react-native-svg": "^12.4.0",
+    "react-native-svg": "^12.4.1",
```

Comment:
Change this version to 12.3.0 to be compatible with expo.

Reply:
done